src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.tsx

```diff
@@ -0,0 +1,144 @@
+      setValue(`${name as string}.scores`, [Math.min(...scores) - 1].concat(scores));
+  }, [min, name, getValues, setValue]);
+
+  useEffect(() => {
+    const { min, scores } = getValues(name);
+
+    const scoresQuantity = max - min + 1;
+
+    if (scores?.length < scoresQuantity)
+      setValue(`${name}.scores`, scores.concat(Math.max(...scores) + 1));
+
+    if (scores?.length > scoresQuantity)
+      setValue(`${name}.scores`, scores?.slice(0, scoresQuantity));
+  }, [max, name, getValues, setValue]);
+
+  const handleCollapse = () => setIsExpanded(!isExpanded);
+
+  return (
+    <StyledSliderPanelContainer
+      in={isExpanded}
+      key={`slider-container-${id}`}
+      collapsedSize="12.8rem"
+      timeout={0}
+    >
+      <Header
+        name={name}
+        label={label}
+        isExpanded={isExpanded}
+        onArrowClick={handleCollapse}
+        onTrashClick={onRemove}
+        isMultiple={isMultiple}
+      />
+      <StyledInputContainer>
+        <InputController
+          control={control}
+          name={`${name}.minLabel`}
+          placeholder={t('minLabel')}
+          maxLength={SLIDER_LABEL_MAX_LENGTH}
+        />
+        <InputController
+          control={control}
+          name={`${name}.maxLabel`}
+          placeholder={t('maxLabel')}
+          maxLength={SLIDER_LABEL_MAX_LENGTH}
+        />
+      </StyledInputContainer>
+      <StyledFlexTopCenter sx={{ p: theme.spacing(2.4, 0.8) }}>
+        <Slider min={min} max={max} />
+      </StyledFlexTopCenter>
+      <StyledInputContainer>
+        <StyledFlexTopCenter sx={{ flexGrow: 1, gap: '1.2rem' }}>
+          <Uploader
+            uiType={UploaderUiType.Secondary}
+            width={5.6}
+            height={5.6}
+            setValue={(val: string) => setValue(`${name as string}.minImage`, val)}
+            getValue={() => watch(`${name}.minImage`) || ''}
+          />
+          <InputController
+            type="number"
+            control={control}
+            name={`${name}.min`}
+            label={t('minValue')}
+            defaultNumberValue={DEFAULT_SLIDER_MIN_NUMBER}
+          />
+        </StyledFlexTopCenter>
+        <StyledFlexTopCenter sx={{ flexGrow: 1, gap: '1.2rem' }}>
+          <Uploader
+            uiType={UploaderUiType.Secondary}
+            width={5.6}
+            height={5.6}
+            setValue={(val: string) => setValue(`${name as string}.maxImage`, val)}
+            getValue={() => watch(`${name}.maxImage`) || ''}
+          />
+          <InputController
+            type="number"
+            control={control}
+            name={`${name}.max`}
+            label={t('maxValue')}
+          />
+        </StyledFlexTopCenter>
+      </StyledInputContainer>
+      <StyledScoresContainer>
+        <Table
+          columns={SLIDER_VALUE_HEAD_ROWS}
+          rows={SLIDER_LABEL_ROWS}
+          orderBy="0"
+          uiType={UiType.Secondary}
+        />
+        <Table
+          columns={getHeadCells(min, max)}
```

Comment:
the score value can be with the minus as I see in the comment of Veronika

Reply:
`getHeadCells` just generates table head cells for Scores' table according to the selected 'Min value' and 'Max value' which cannot be less then 0, so there isn't potential problems here

Reply:
> the score value can be with the minus as I see in the comment of Veronika
> 
> the score value is overlapping by arrows in the 16 inch laptop, pls fix ![image](https://user-images.githubusercontent.com/116339462/224952144-015ecab1-7ffd-4262-914b-22ae4679e42f.png)

done